prac2_login_verification_UI/main.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MaterialApp(
+  home: Loginpage(),
+));
+
+class Loginpage extends StatefulWidget {
+  const Loginpage({Key? key}) : super(key: key);
+
+  @override
+  State<Loginpage> createState() => _LoginpageState();
+}
+
+class _LoginpageState extends State<Loginpage> {
+  bool value=true;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Padding(
+        padding: EdgeInsets.fromLTRB(30.0, 100.0, 30.0, 0.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Center(
+              child: Text(
+                'Login',
+                style: TextStyle(
+                  fontSize: 26.0,
+                  fontWeight: FontWeight.bold,
+                  // letterSpacing: 2.0,
+                ),
+              ),
+            ),
+            SizedBox(height: 20.0),
+            Center(
+              child: Text(
+                'Create an account',
+                style: TextStyle(
+                    color: Colors.lightBlue[800],
+                    fontSize: 16.0
+                ),
+              ),
+            ),
+            SizedBox(height: 30.0),
+
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Text(
+                  'Username',
+                  style: TextStyle(
+                      fontSize: 24.0,
+                      fontWeight: FontWeight.bold
+                  ),
+                ),
+                SizedBox(height: 10.0),
+                Container(
+                  decoration: BoxDecoration(
+                    border: Border.all(color: Colors.grey),
+                    borderRadius: BorderRadius.circular(12.0),
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(20.0,0,0,0),
+                    child: TextField(
+                      decoration: InputDecoration(
+                          hintText: 'Enter your username',
+                          contentPadding: EdgeInsets.symmetric(vertical: 20.0)
+                      ),
+                    ),
+                  ),
+                ),
+                SizedBox(height: 30.0),
+                Text(
+                  'Password',
+                  style: TextStyle(
+                      fontSize: 24.0,
+                      fontWeight: FontWeight.bold
+                  ),
+                ),
+                SizedBox(height: 10.0),
+                Container(
+                  decoration: BoxDecoration(
+                    border: Border.all(color: Colors.grey),
+                    borderRadius: BorderRadius.circular(12.0),
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(20.0,0,0,0),
+                    child: TextField(

```

Comment:
We would like to hide away the text input by user in a password field, we should see *** instead of 'abc'.
Please make this fix in subsequent commits :100:

Reply:
Resolved in 65e2e0c.... :+1: